new/PRS12787GabraEgziabeher.xml

```diff
@@ -30,7 +30,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p/>
+                <p>Gabra ʾƎgziʾabǝḥer is mentioned as
```

Comment:
I would add this information either as a desc in the relation, or in occupation

Reply:
Ok